### DISCUSSION TITLE
Multiple Mesh Files Import

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#25801] [Q&A Meshing]
I am wanting to use multiple mesh files in the contact modeling module. The mesh domains are not required to be stitched but have to be separated to control individual file sized. Can multiple .e files be opened with FileMesh?

**Comment by GiudGiud** [comment]
Not with file mesh but with FileMeshGenerator you can do that with a generator for each mesh file

**Reply by GiudGiud** [reply]
then use a CombineMeshGenerator to combine them without stitiching
